README.md

```diff
@@ -141,49 +177,41 @@ the snippet above with `<arg value="-F"/>`.
 
 To run diktat to check/fix code style - run `$ mvn antrun:run@diktat`.
 
-### Use the new diktat-maven-plugin
+## Run with Gradle
+### Use diktat-gradle-plugin
+You can see how the plugin is configured in our project for self-checks: [build.gradle.kts](build.gradle.kts).
+Add this plugin to your `build.gradle.kts`:
+```kotlin
+plugins {
+    id("org.cqfn.diktat.diktat-gradle-plugin") version "0.1.4"
+}
+```
 
-You can see how it is configured in our project for self-checks: [pom.xml](pom.xml).
-This plugin should be available since version 0.1.3. It requires less configuration but may contain bugs.
-If you use it and encounter any problems, feel free to open issues on [github](https://github.com/cqfn/diktat/issues).
+Or use buildscript syntax:
+```kotlin
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath("org.cqfn.diktat:diktat-gradle-plugin:0.1.4")
+    }
+}
 
-Add this plugin to your pom.xml:
-<details>
-  <summary><b>Maven plugin snippet</b></summary><br>
-  
-```xml
-            <plugin>
-                <groupId>org.cqfn.diktat</groupId>
-                <artifactId>diktat-maven-plugin</artifactId>
-                <version>${diktat.version}</version>
-                <executions>
-                    <execution>
-                        <id>diktat</id>
-                        <phase>none</phase>
-                        <goals>
-                            <goal>check</goal>
-                            <goal>fix</goal>
-                        </goals>
-                        <configuration>
-                            <inputs>
-                                <input>${project.basedir}/src/main/kotlin</input>
-                                <input>${project.basedir}/src/test/kotlin</input>
-                            </inputs>
-                            <diktatConfigFile>diktat-analysis.yml</diktatConfigFile>
-                        </configuration>
-                    </execution>
-                </executions>
-            </plugin>
+apply(plugin = "org.cqfn.diktat.diktat-gradle-plugin")
 ```
 
-</details>
-
-To run diktat check use command `$ mvn diktat:check@diktat`.
-To run diktat in autocorrect mode use command `$ mvn diktat:fix@diktat`.
+You can then configure diktat using `diktat` extension:
```

Comment:
list of available option? like `input`. Any other?

Reply:
For now only `debug`, I'll add it to readme.